flow/probes/ebpf.go

```diff
@@ -77,6 +76,8 @@ func (p *EBPFProbe) run() {
 	_, flowEBPFChan, _ := p.flowTable.Start()
 	defer p.flowTable.Stop()
 
+	ebpfUpdate := time.Duration(config.GetConfig().GetInt("agent.flow.ebpf_kernel_scan_interval")) * time.Second
```

Comment:
can you add an example entry in the default config file etc/skydive.yml.default. We use per probe section, so it should be `agent.flow.ebpf.kernel_scan_interval`

Reply:
done